script/release-artifact-cleanup.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+if (!process.env.CI) require('dotenv-safe').load()
+require('colors')
+const args = require('minimist')(process.argv.slice(2), {
+  boolean: ['tag']
+})
+const { execSync } = require('child_process')
+const { GitProcess } = require('dugite')
+
+const GitHub = require('github')
+const path = require('path')
+
+const github = new GitHub()
+const gitDir = path.resolve(__dirname, '..')
+
+github.authenticate({
+  type: 'token',
+  token: process.env.ELECTRON_GITHUB_TOKEN
+})
+
+function getLastBumpCommit () {
+  const data = execSync(`git log - n1--grep "Bump v[0-9.]*"--format = "format:{hash: %H, message: '%s'}"`)
+  return JSON.parse(data)
```

Comment:
Can we use `GitProcess.exec()` for this so that we can have proper error-checking?